roles/nodejs/tasks/main.yml

```diff
@@ -57,54 +24,53 @@
   register: yarnmit_servers
   ignore_errors: true
 
-- name: Add upstream Yarn APT key from PGP
+- name: nodejs | Add upstream Yarn APT key from PGP
   apt_key:
     id: "{{ nodejs__yarn_upstream_key_id }}"
     state: present
     keyserver: "keyserver.pgp.com"
   when: yarnmit_servers is failed
```

Comment:
Hm. This will run when the MIT key is missing but we already have a key from sks. Another reason to put this in a loop if we can.

Reply:
thanks!